config/longrun_configs/amip_target.yml

```diff
@@ -1,18 +1,23 @@
-run_name: "amip_target"
 anim: true
```

Comment:
the alphabetization is really nice!! it would be nice to have this standardized across all the yml files too (but we can do that in a different PR)

Reply:
I was going to do this as part of a separate PR right after this, so that I don't overwhelm you guys with too many files to review at once. :)

Reply:
https://github.com/CliMA/ClimaCoupler.jl/issues/595